FxAClient/Frontend/SignIn/FxAContentViewController.swift

```diff
@@ -112,6 +114,20 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
         self.timer?.invalidate()
         self.timer = nil
         self.isLoaded = true
+        
+        fillField("email", value: self.fxaOptions.email)
+        fillField("access_code", value: self.fxaOptions.access_code)
+    }
+    
+    // Attempt to fill out a field in content view
+    private func fillField(className: String?, value: String?){
+        if(className == nil || value == nil){
```

Comment:
We usually prefer guard but tbh in this case I think either is fine. Make sure to add a space between the `)` and `{`